test/rekt/channel_test.go

```diff
@@ -39,5 +39,48 @@ func TestChannelConformance(t *testing.T) {
 		environment.Managed(t),
 	)
 
-	env.TestSet(ctx, t, channel.ControlPlaneConformance())
+	channelName := "mychannelimpl"
+
+	// Install and wait for a Ready Broker.
```

Comment:
update cut&paste?

Reply:
ooops!! nice catch, will fix in the next PR.